components/Info/Info.tsx

```diff
@@ -0,0 +1,20 @@
+import styles from "../Info/Info.module.css";
+export type InfoProps = {
+  firstWord: string;
+  otherWords: string;
+  information: string;
+};
+
+function Info({ firstWord, otherWords, information }: InfoProps) {
```

Comment:
`firstWord`, `otherWords` and `information` isn't perfect naming. The prop names should express what is inside them